composeApp/src/commonMain/kotlin/com/github/familyvault/backend/models/ThreadPrivateMeta.kt

```diff
@@ -3,4 +3,4 @@ package com.github.familyvault.backend.models
 import kotlinx.serialization.Serializable
 
 @Serializable
-data class ThreadPrivateMeta(val name: String, val referenceStoreId: String?)
\ No newline at end of file
+data class ThreadPrivateMeta(val name: String, val referenceStoreId: String?, val managersPublicKeys: List<String>?)
```

Comment:
Zmieniłbym tą nazwę na `initialManagersPublicKeys` aby rozróżnić to między listą managerów PrivMx